updater/aws_test.go

```diff
@@ -189,10 +192,11 @@ func TestSendCommandWaitSuccess(t *testing.T) {
 		commandID, err := u.sendCommand(instances, "test-doc")
 		require.NoError(t, err)
 		assert.Equal(t, "command-id", commandID)
-		assert.Equal(t, expectedFailInstances, failedInstanceIDs, "should match instances for which wait fail")
+		assert.Contains(t, expectedFailInstances, "inst-id-1", "should match instance for which wait fails")
```

Comment:
According to shailesh's suggestion above, if we can use `ElementsMatch` here as well?  :slightly_smiling_face: